src/Sylius/Component/Core/spec/Promotion/Action/UnitFixedDiscountPromotionActionCommandSpec.php

```diff
@@ -194,6 +236,78 @@ function it_does_not_apply_bigger_discount_than_unit_total(
         $this->execute($order, ['WEB_US' => ['amount' => 1000]], $promotion)->shouldReturn(true);
     }
 
+    function it_does_not_apply_bigger_discount_than_variant_minimum_price(
+        ChannelInterface $channel,
+        FactoryInterface $adjustmentFactory,
+        FilterInterface $priceRangeFilter,
+        FilterInterface $taxonFilter,
+        FilterInterface $productFilter,
+        AdjustmentInterface $promotionAdjustment1,
+        AdjustmentInterface $promotionAdjustment2,
+        OrderInterface $order,
+        OrderItemInterface $orderItem,
+        OrderItemUnitInterface $unit1,
+        OrderItemUnitInterface $unit2,
+        OrderItemInterface $orderItem1,
+        OrderItemInterface $orderItem2,
+        ProductVariantInterface $productVariant1,
+        ProductVariantInterface $productVariant2,
+        PromotionInterface $promotion,
+        ChannelPricingInterface $channelPricing1,
+        ChannelPricingInterface $channelPricing2
+    ): void {
+        $order->getChannel()->willReturn($channel);
+        $channel->getCode()->willReturn('WEB_US');
+
+        $unit1->getOrderItem()->willReturn($orderItem1);
+        $unit2->getOrderItem()->willReturn($orderItem2);
+
+        $orderItem1->getVariant()->willReturn($productVariant1);
+        $orderItem2->getVariant()->willReturn($productVariant2);
+        $orderItem1->getOrder()->willReturn($order);
+        $orderItem2->getOrder()->willReturn($order);
+
+        $productVariant1->getChannelPricingForChannel($channel)->willReturn($channelPricing1);
+        $productVariant2->getChannelPricingForChannel($channel)->willReturn($channelPricing2);
+
+        $channelPricing1->getMinimumPrice()->willReturn(200);
+        $channelPricing2->getMinimumPrice()->willReturn(150);
+
+        $order->getItems()->willReturn(new ArrayCollection([$orderItem]));
```

Comment:
Something is wrong with this spec. We have an order with one item of quantity two, but its two units return second and third order items, while called `$orderItem1` and `$orderItem2` to make it even more confusing. Can we just define order with two order items?